### PR TITLE
Qhull update v2020.2

### PR DESCRIPTION
**Pull request by nieder**
newest release for Qhull
Only patch needed was to force an absolute install_name on library.

### CODE REVIEW

**Comment by nieder** [comment]
Ugh. The qhull pc files say the library versions are 8.0.2, but upstream is using the year.minor nomenclature for the package versioning. And the executables report both:
```
$ ./qhull -V
qhull_r 8.0.2 (2020.2.r 2020/08/31)
```
Ubuntu and MP use the year nomenclature.

**Review by akhansen — DISMISSED** [review]
Looks fine to me.  

Go ahead and take me off as maintainer, too, since I don't use it.

**Comment by nieder** [comment]
@dmacks for qhull8 (executables), just rename it to qhull? Any C/R needed?

**Comment by nieder** [comment]
Ready for review

File: 10.9-libcxx/stable/main/finkinfo/sci/qhull8.0-shlibs.info [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+Shlibs: <<
+	%p/lib/libqhull_r.8.0.dylib 8.0.0 %n (>= 2020.2-1)
+<<
+
+DocFiles: Announce.txt COPYING.txt README.txt REGISTER.txt
+Homepage:  http://www.qhull.org/
+SplitOff:  <<
+	Package: qhull
+	Conflicts: qhull, qhull5, qhull6, qhull6.3, qhull6.3.1, qhull
+	Replaces: qhull, qhull5, qhull6, qhull6.3, qhull6.3.1, qhull
```
Comment (dmacks) [inline 10.9-libcxx/stable/main/finkinfo/sci/qhull8.0-shlibs.info:83]: "qhull" is listed twice.

File: 10.9-libcxx/stable/main/finkinfo/sci/qhull8.0-shlibs.info [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	Files: <<
+		bin
+		share/doc/qhull
+		share/man
+	<<
+	DocFiles: Announce.txt COPYING.txt README.txt REGISTER.txt index.htm
+	DescUsage: <<
+	This package contains the executables, manpages, and other documentation.
```
Comment (dmacks) [inline 10.9-libcxx/stable/main/finkinfo/sci/qhull8.0-shlibs.info:91]: Might want to comment on the unusual situation of a binaries pkg not having dependencies on its own -shlibs (I guess it's static-linked?)

File: 10.9-libcxx/stable/main/finkinfo/sci/qhull6.3.1.info [hunk 3]
```diff
@@ -1,9 +1,9 @@
 Package: qhull6.3.1
```
Comment (dmacks) [inline 10.9-libcxx/stable/main/finkinfo/sci/qhull6.3.1.info:1]: Wanna turn this .info layout inside-out also, so we can kill this libversion easily soon?

**Review by dmacks — DISMISSED** [review]

File: 10.9-libcxx/stable/main/finkinfo/sci/qhull6-shlibs.info [hunk 4]
```diff
@@ -31,11 +36,10 @@ Source: http://www.qhull.org/download/qhull-%v-src.tgz
...
 SourceDirectory: qhull-%v
 
-Depends: %N-shlibs (= %v-%r )
-Conflicts: qhull, qhull5, qhull6.3, qhull6.3.1
-Replaces: qhull, qhull5, qhull6.3, qhull6.3.1
+Conflicts: qhull6-dev (<< 2011.2-5) 
+Replaces: qhull6-dev (<< 2011.2-5) 
 
-PatchFile: %n.patch
+PatchFile: qhull6.patch
```
Comment (dmacks) [inline 10.9-libcxx/stable/main/finkinfo/sci/qhull6-shlibs.info:42]: Probably best to keep the .patch in sync with the .info

File: 10.9-libcxx/stable/main/finkinfo/sci/qhull8.0-shlibs.info (hunk 2 above)
Comment (nieder) [reply 10.9-libcxx/stable/main/finkinfo/sci/qhull8.0-shlibs.info:89]: In `DescPackaging`:
```
* Upstream has decided to build the executables statically, so Depends:
  %N is not required.
```
Moved the `qhull` package to remove ambiguity.

File: 10.9-libcxx/stable/main/finkinfo/sci/qhull8.0-shlibs.info (hunk 1 above)
Comment (nieder) [reply 10.9-libcxx/stable/main/finkinfo/sci/qhull8.0-shlibs.info]: See 5028062

**Comment by dmacks** [comment]
There's going to be a deadlock if anyone has [qhull5 qhull5-shlibs] installed and tries to update qhull5-shlibs (qhull5 no longer extant) because the qhull5:Depends:qhull5-shlibs dependency is "= %v-%r". So qhull5 has to be removed first (either directly, or by installing the new qhull that C/R it). This is a common problem when a binary:depends:-shlibs is "=" (using ">=" prevents the problem). Brainstorming...should qhull5-shlibs:C/R:qhull5 (same as it C/R qhull5-dev)? Likewise with qhull6.

Sorry I didn't notice the weird-bin-dependency DescPackaging note!

File: 10.9-libcxx/stable/main/finkinfo/sci/qhull6.3.1.info (hunk 3 above)
Comment (nieder) [reply 10.9-libcxx/stable/main/finkinfo/sci/qhull6.3.1.info]: Done

File: 10.9-libcxx/stable/main/finkinfo/sci/qhull6-shlibs.info (hunk 4 above)
Comment (nieder) [reply 10.9-libcxx/stable/main/finkinfo/sci/qhull6-shlibs.info]: Done

**Review by dmacks — APPROVED** [review]
lgtm; any further improvements can happen later.

**Pull request merged**